packaging/osx/sharedframework/shared-framework-distribution-template.xml

```diff
@@ -26,4 +26,19 @@
     <pkg-ref id="{SharedFxComponentId}.pkg">{SharedFxComponentId}.pkg</pkg-ref>
     <pkg-ref id="{HostFxrComponentId}.pkg">{HostFxrComponentId}.pkg</pkg-ref>
     <pkg-ref id="{SharedHostComponentId}.pkg">{SharedHostComponentId}.pkg</pkg-ref>
+    <installation-check script="openSSLCheck()" />
+    <script><![CDATA[
+        function openSSLCheck()
+        {
+            var openSSLRoot = "/usr/local/opt/openssl/lib/";
```

Comment:
@blackdwarf I don't authoritatively.  https://github.com/dotnet/corefx/issues/9171#issuecomment-226950619 suggests that maybe it puts it in `/opt/local/lib`.